### ISSUE TITLE
Wrong image section in WebP for "manuelly" cropped images

### ISSUE THREAD

**Issue by scrobbleme** [nosilver4u/ewww-image-optimizer#348]
**Describe the bug**
We are using the plugin [Crop Thumbnails](https://de.wordpress.org/plugins/crop-thumbnails/) to manually select the correct image section for thumbnails.

Unfortunately, since the latest EWWW update, the WebP generated images are always "centered"

I think this can be related to:

>  added: improved WebP Conversion quality by using full-size/original source for thumbs

For us, it would be fine to have a manual flag or filter to disable this new functionality and just use the generated size, as before.

**Comment by nosilver4u** [comment]
Yup, totally expected we would run into some things like that! You can use the filter 'ewwwio_use_original_for_webp_thumbs' to go back to the previous behavior. Something like this should do it:
`add_filter( 'ewwwio_use_original_for_webp_thumbs', '__return_false' );
`

**Comment by nosilver4u** [comment]
FYI, I've already committed a fix to do this automatically when Crop Thumbnails is installed if you want to grab the latest from Github.
